recommend/html/record.html

```diff
@@ -6,7 +6,11 @@
 						</td>
 					{{else if eq $i 2}}
 						<td>
-						{{.Name}}/10
+						{{if eq .Name "-"}}
+							{{.Name}}
+						{{else}}
+							{{.Name}}/10
+						{{end}}
```

Comment:
Kyverno policies don't have any severity right now, we have not defined a level of severity for them. They right now have severity `-`, so `-/10` looked weird, so when `severity == "-"`, just display the severity as `-` instead of `-/10`